### PR TITLE
add version label to property rules

### PR DESCRIPTION
**Pull request by phil198**
add [role=label--new-5.21] version label to property rules

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1818.surge.sh

Updated pages:
- [Property-based access control](https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1818.surge.sh

Updated pages:
- [Property-based access control](https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1818.surge.sh

Updated pages:
- [Property-based access control](https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control)

File: modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ CREATE ROLE regularUsers;
 ////
 
 [role=enterprise-edition aura-db-enterprise]
```
Comment (renetapopova) [inline modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc:10]:
```suggestion
[role=enterprise-edition aura-db-enterprise label--new-5.24]
```

File: modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc [hunk 2]
```diff
@@ -8,6 +8,7 @@ CREATE ROLE regularUsers;
...
+[role=label--new-5.24]
```
Comment (renetapopova) [inline modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc:11]:
```suggestion
```

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @phil198. I think they should be part of the same attribute to be rendered.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1818.surge.sh

Updated pages:
- [Property-based access control](https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control)

**Comment by phil198** [comment]
thanks @renetapopova . For some reason it still doesn't render https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control/. Any ideas?

File: modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ CREATE ROLE regularUsers;
 ----
 ////
 
-[role=enterprise-edition aura-db-enterprise]
+[role=enterprise-edition aura-db-enterprise label--new-5.24]
```
Comment (renetapopova) [inline modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc:10]:
```suggestion
:page-role: enterprise-edition aura-db-enterprise new-5.24
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1818.surge.sh

Updated pages:
- [Property-based access control](https://neo4j-docs-operations-1818.surge.sh/operations-manual/5/authentication-authorization/property-based-access-control)

**Review by renetapopova — APPROVED** [review]
I think it looks good now.

**Comment by phil198** [comment]
> I think it looks good now.

Thanks @renetapopova

**Pull request merged**